src/D2L.CodeStyle.Analyzers/ApiUsage/JsonParamBinderAttribute/JsonParamBinderAnalyzer.cs

```diff
@@ -85,16 +85,16 @@ TypeWhitelist whitelist
 			context.ReportDiagnostic( diagnostic );
 		}
 
-		private void PreventUnnecessaryWhitelisting(
+		private void PreventUnnecessaryAllowedListing(

```

Comment:
I especially don't love `AllowedListing` but I'm not sure what verb would be more appropriate. `PreventUnnecessaryAllowing` seems not great.

Reply:
PreventUnnecessaryAllowedEntry? meh.
Ironically "white" here means bad :)

"Permit" came to mind, like you get a permit to do whatever bad thing we're trying to prevent. I saw other people float "Exception" but that's bad for reasons (ambiguity in C#). Exemption is a different word but kinda looks the same as Exception.

I don't know, I don't have any feelings. I think what you've done is fine.

Reply:
Yeah, that's fair. Words are hard. :P